Model/Tax/Sales/Total/Quote/Tax.php

```diff
@@ -174,7 +174,7 @@ public function collect(
         // their address, such as on the product listing, product detail, or cart page. Once the user enters their
         // postcode in the "Estimate Shipping & Tax" form on the cart page, or submits their shipping address in the
         // checkout, then a postcode will be present.
-        if (!$postcode) {
+        if (!$postcode || \count($postcode) < 4) {
```

Comment:
@sjb9774 - I think what you meant to use here was `\strlen` instead of `\count`.